src/particles/webgl/ParticleRenderer.js

```diff
@@ -153,6 +154,7 @@ export default class ParticleRenderer extends core.ObjectRenderer
 
         this.shader.uniforms.projectionMatrix = m.toArray(true);
         this.shader.uniforms.uAlpha = container.worldAlpha;
+        this.shader.uniforms.tint = hex2rgb(container.tint);
```

Comment:
`hex2rgb` creates a new Array every `render` call here, which isn't good performance-wise. Maybe consider adding a private `_tintRGB` property on the ParticleContainer when `tint` is set? 

```js
set tint(value)
{
     this._tint = value;
     hex2rgb(value, this._tintRGB);
}
```
And where this is added to the constructor before the initial `this.tint = 0xFFFFFF`.
```js
this._tintRGB = []; 
```

Then this line could be `this.shader.uniforms.int = container._tintRGB`